packages/contracts-bedrock/.storage-layout

```diff
@@ -24,8 +24,8 @@
 | xDomainMsgSender   | address                   | 204  | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
 | msgNonce           | uint240                   | 205  | 0      | 30    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
 | failedMessages     | mapping(bytes32 => bool)  | 206  | 0      | 32    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| __gap              | uint256[42]               | 207  | 0      | 1344  | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| superchainConfig   | contract SuperchainConfig | 249  | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| __gap              | uint256[44]               | 207  | 0      | 1408  | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
+| superchainConfig   | contract SuperchainConfig | 251  | 0      | 20    | src/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
```

Comment:
did you mean to make it 251 and not 250?

Reply:
Yes, here's a comment on the outdated code: https://github.com/ethereum-optimism/optimism/pull/8345#discussion_r1409809328